zerver/lib/cache.py

```diff
@@ -69,18 +76,21 @@ def get_or_create_key_prefix():
 
     return prefix
 
-KEY_PREFIX = get_or_create_key_prefix()
+KEY_PREFIX = get_or_create_key_prefix() # type: str
 
 def bounce_key_prefix_for_testing(test_name):
+    # type: (str) -> None
     global KEY_PREFIX
     KEY_PREFIX = test_name + ':' + str(os.getpid()) + ':'
 
 def get_cache_backend(cache_name):
+    # type: (str) -> Union[get_cache, djcache]
```

Comment:
I those those both subclass from the same thing, so we might be able to do a single class here as the return type